.github/workflows/validate-release-notes.yml

```diff
@@ -0,0 +1,17 @@
+name: Validate Release Notes
+
+on:
+  pull_request:
+    branches: [main]
+    paths: ["release-notes/*.yml"]
+    
+jobs:
+  validate:
+    name: Validate Release Notes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
Since this is new work, we might as well use the latest version.

Reply:
changed it to v3